README.md

```diff
@@ -1,6 +1,6 @@
 # CLDF dataset derived from Z'graggen's "Madang Comparative Wordlists" from 1980
 
-[![CLDF validation](https://github.com/lexibank/zgraggenmadang/workflows/CLDF-validation/badge.svg)](https://github.com/lexibank/zgraggenmadang/actions?query=workflow%3ACLDF-validation)
+[![CLDF validation](https://github.com/MuffinLinwist/zgraggenmadang/workflows/CLDF-validation/badge.svg)](https://github.com/MuffinLinwist/zgraggenmadang/actions?query=workflow%3ACLDF-validation)
```

Comment:
@chrzyki, thanks for pointing this out! Didn't think of it. How can I set the CLDF validation to the repo on Lexibank from a forked repo? Or is it not possible?

Reply:
Maybe receiving write permission from the organization would be a better solution since I need to fix some Glottocodes from several datasets for the new release of Lexibank, as indicated in [lexibank/lexibank-analysed/issues/47](https://github.com/lexibank/lexibank-analysed/issues/47).